### DISCUSSION TITLE
Is there any examples for RF 315/433Mhz ?

### DISCUSSION THREAD

**Discussion by GF-Huang** [gpiozero/gpiozero#939] [Q&A]
I'm using `MX-05V Receiver` and `MX-FS-03V Transmitter`.

![image](https://user-images.githubusercontent.com/4510984/103437966-9f6d9200-4c68-11eb-8a5f-4d58b62f8007.png)

https://www.geekbuying.com/item/MX-05V-433MHZ-RF-Wireless-Receiver-Module-MX-FS-03V-Transmitter-Module-Kit-For-Arduino-Raspberry-Pi-AVR-ARM-344591.html

**Comment by lurch** [comment]
Nope, not currently any support for that in GpioZero.

There is support for https://gpiozero.readthedocs.io/en/stable/api_boards.html#energenie but that's slightly different :wink:

**Comment by bennuttall** [comment]
If you have some working code for another library like RPi.GPIO that would be handy.
